### ISSUE TITLE
Text containing links is not underlined, affecting accessibility

### ISSUE THREAD

**Issue by cristinavaldera** [exelearning/exelearning#1215]
Text that contains links is not displayed as underlined, which makes it harder to identify, particularly from an accessibility perspective. According to accessibility best practices, links should be clearly distinguishable from regular text, and underlining is a key visual indicator, especially for users with color perception difficulties.

**Comment by RRomanillos15** [comment]
Hola Franmate,

Nosotros intentamos **sin éxito** aplicarlo en el estilo CSS de la organización. De hecho, si os fijáis....eso pasa también en todos los estilos eXe predeterminados de la herramienta en su versión eXe 3.0 (Preestablecido, Zen, Neo, Flux...). 

<img width="1842" height="696" alt="Image" src="https://github.com/user-attachments/assets/2a68c650-21b5-426d-8690-8fcf4be8633d" />

<img width="1881" height="600" alt="Image" src="https://github.com/user-attachments/assets/cf2f87c6-65fb-4548-a980-552f13897ae3" />
